torch/quantization/_quantize_script.py

```diff
@@ -76,6 +76,17 @@ def prepare_script(model, qconfig_dict, inplace=False):
                                                                 False))
     return model
 
+def prepare_dynamic_script(model, qconfig_dict, inplace=False):
+    _check_is_script_module(model)
+    if not inplace:
```

Comment:
also we don't want to have inplace flag, you can remove it now

Reply:
we'll have a separate PR to remove all inplace flag for insert observers, I haven't got time to do that

Reply:
Will you be removing it from the pass API as well?

Reply:
yes

Reply:
So is inplace true or false by default?

Reply:
I mean we'll revisit after the `clone` API is ready, current inplace doesn't really make sense and should be removed

Reply:
`inplace` should be False

Reply:
qq why do we have `model = model.copy()` here? seems redundant.